### PR TITLE
GetFileInfo: add page

### LINKED ISSUE #3993: page request: GetFileInfo
Used to get the information of a file in macOS

https://ss64.com/osx/getfileinfo.html

### PR DESCRIPTION
**Pull request by rachithaiyappa**
Closes #3993.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/GetFileInfo.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
-
 # GetFileInfo
 
-> Get information about a file in an HFS+ directory
+> Get information about a file in an HFS+ directory.
 
-- Date of file creation
+- Date of file creation:
```
Comment (zdroid) [inline pages/osx/GetFileInfo.md:5]:
```suggestion
- See date of file creation:
```

File: pages/osx/GetFileInfo.md [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 
 `GetFileInfo -d {{filename}}`
 
-- Date of latest file modification
+- Date of latest file modification:
```
Comment (zdroid) [inline pages/osx/GetFileInfo.md:9]:
```suggestion
- See date of latest file modification:
```

File: pages/osx/GetFileInfo.md [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
 
 `GetFileInfo -m {{filename}}`
 
-- Creator of the file
+- Creator of the file:
```
Comment (zdroid) [inline pages/osx/GetFileInfo.md:13]:
```suggestion
- See creator of the file:
```

File: pages/osx/GetFileInfo.md [hunk 4]
```diff
@@ -1,21 +1,19 @@
...
 
 `GetFileInfo -c {{filename}}`
 
-- All infomation about the file
+- All infomation about the file:
```
Comment (zdroid) [inline pages/osx/GetFileInfo.md:17]:
```suggestion
- See all infomation about the file:
```

**Comment by zdroid** [comment]
I edited your PR's description to note it closes #3993.

**Comment by rachithaiyappa** [comment]
@zdroid, thanks for the suggested changes. I've implemented them in the latest commit. Do have a look.

I'm not really sure why license/cla is pending. The block before I re-opened the pull request shows rachithaiyappa (which is me) has signed it but not sure how Rachith Aiyappa (this is not my username) came in over there. What do you suggest?

**Comment by zdroid** [comment]
Oh, I didn't notice that. It's pending because your first commit was made without authentication to your account. If you don't want to rewrite Git history, the easiest thing to do is to open a new PR in which you commit properly.

**Comment by rachithaiyappa** [comment]
@zdroid, I'll just close this and open a new pull request. Hope that's okay

**Comment by zdroid** [comment]
It's okay, but you need to make it on a new branch. Otherwise the problematic commit will appear in PR again.

**Comment by rachithaiyappa** [comment]
To tackle the issue of license and not wanting to rewrite git history, I have closed this PR and opened a new one at https://github.com/tldr-pages/tldr/pull/4004

**Pull request closed without merging**